dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -60,6 +60,16 @@ kafka.library.not.present.CWMRX1006E=CWMRX1006E: Classes from the kafka-clients
 kafka.library.not.present.CWMRX1006E.explanation=Applications that use the Reactive Messaging Kafka Client must provide the kafka-clients JAR file and its dependencies. They can provide this file and its dependencies either by packaging it as a library within the application or by providing it as a shared library.
 kafka.library.not.present.CWMRX1006E.useraction=Package the kafka-clients JAR file and its dependencies as libraries in your application or as a shared library. The kafka-clients JAR file can be downloaded from the Apache Kafka website or from Maven Central.
 
+# Could not create a Kafka incoming connector for the {0} channel. The error is {1}
```

Comment:
Change,
" Could not create a Kafka incoming connector for the {0} channel. The error is {1}"
to,
" A Kafka incoming connector for the {0} channel cannot be created"

Reply:
I've kept " The error is {1}" since part of the goal of this PR was to ensure that the underlying error message is exposed to the user. I've replaced the first part of the message with what you suggested.